kolibri/core/exams/serializers.py

```diff
@@ -74,6 +75,7 @@ class Meta:
             'id', 'title', 'channel_id', 'question_count', 'question_sources', 'seed',
             'active', 'score', 'archive', 'answer_count', 'closed',
         )
+        fields = '__all__'
```

Comment:
Could this be left undefined? Just to enforce that it gets explicitly defined in inheritors?

Reply:
No, this has to be defined, which is why I added it.